packages/@okta/vuepress-site/docs/guides/password-import-hook/check-authorization-header/nodejs/check-authorization-header.md

```diff
@@ -0,0 +1,14 @@
+For the check of the authorization in header here, we use Node packages to parse the request from Okta and to decode the Authorization header.
+
+We're going to choose to use the Basic Authentication scheme to encode a username and password combination in base64 to serve as the content of the Authorization header.
```

Comment:
Should also mention Basic Authentication is used here for only for demonstration purposes and that our customers should use a stronger form of Authentication.

Reply:
As discussed, there is only support for setting the Authorization header to a static string.